ci.cue

```diff
@@ -87,6 +89,43 @@ dagger.#Plan & {
 			command: flags: "-race": true
 		}
 
+		integration: bats.#Bats & {
+			_daggerLinuxBin: build.go & {
+				os: "linux"
```

Comment:
Why linux?

Reply:
Because this bin runs inside a linux container, but same arch than the host. What would you set?

Reply:
I would set nothing for both os and arch, leave the default of `go.#Build` which will use buildkit os/arch (e.g. will also work on windows)